models/DataObject/Data/QuantityValueRange.php

```diff
@@ -62,12 +62,12 @@ public function setMaximum(int|float|null $maximum): void
         $this->markMeDirty();
     }
 
-    public function getRange(int $step = 1): array
+    public function getRange(float $step = 0.1): array
```

Comment:
mhhh looks bc-breaky, as the declaration would be incompatible.

Should we keep this and add a new method `getDecimalRange(float $step = 0.1): array`? 
For `getValue()` it would be more tricky

Reply:
Yeah I was not sure either. I tested it and nothing seemed to change though.
I don't really know the implications of changing the step-size here.

Reply:
ok, it looks like it was never passed a `$step` value other than the default.
There are no other ways to fix it without breaking a the signature, the impact of potential bc break seems way less  than keeping it as broken (as it wouldn't work with decimals unless is the same for both), so resolving this point